src/blockchain.js

```diff
@@ -156,10 +156,13 @@ Blockchain.doConnect = function(connectionList, opts, doneCb) {
           self.blockchainConnector.setDefaultAccount(accounts[0]);
         }
 
-        const connectionError = new BlockchainConnectionError(connectionErrs);
+        let connectionError;
+        if (!result.connected || result.error) {
```

Comment:
Should we put this before line 141 (`self.blockchainConnector.getAccounts(`)? It seems to make sense that if we can't connect to the blockchain, that we should show a connection error before attempting to get accounts from the node.

Reply:
Makes sense.